src/dotnet-scaffolding/dotnet-scaffold-aspnet/Common/ModelInfo.cs

```diff
@@ -10,6 +10,8 @@ internal class ModelInfo
     public List<IPropertySymbol>? ModelProperties { get; set; }
     public string? ModelNamespace { get; set; }
     public string ModelTypeName { get; set; } = default!;
+    public string ModelTypeNameCapitalized => char.ToUpper(ModelTypeName[0]) + ModelTypeName.Substring(1);
```

Comment:
Is `ModelTypeName[0]` guaranteed to not be empty? If not, this will throw

Reply:
adding a string.IsNullOrEmpty check, will default to just returning the `ModelTypeName` if empty.